lib/impl/editing.dart

```diff
@@ -10,11 +10,6 @@ import '../atom.dart';
 
 final Logger _logger = new Logger('editing');
 
-// TODO: If we're in a line comment, and the line is longer than the max line
-// length, extend the comment.
-// var prefLineLength = atom.config.get('editor.preferredLineLength',
```

Comment:
Let's keep this TODO: My idea here is that if the comment line (`//`) is longer than 80 cols, and they hit enter, that we should backtrack back to the first ws before 80 cols, wrap the line, and continue the comment. So they can write long lines w/o having to manually know when to wrap a line.